### ISSUE TITLE
Training script for human-mix and tulu mix?

### ISSUE THREAD

**Issue by macabdul9** [allenai/open-instruct#28]
The **`scripts/prepare_train_data.sh`** desn't create the mixture. Do you just concatenate the data? Can you add the training script to for human-mix and tulu mix data. `finetune_with_accelerate.sh` or `finetune_with_accelerate.sh ` is for individual data it seems like.

**Comment by yizhongw** [comment]
Hi @macabdul9 , yeah you can simply concatenate the datasets because they are in the same format. The data loader will randomly shuffle them during training.
